internal/connection_repository/stale_connection_locator.go

```diff
@@ -41,11 +41,12 @@ func ProcessStaleConnections(ctx context.Context, databaseConn *sql.DB, sqlTimeo
 
 	for rows.Next() {
 		var account domain.AccountID
+		var orgID domain.OrgID
```

Comment:
For now, the type of the org_id passed to the `rows.Scan()` will need to be a `sql.NullString` because the org_id can be null in the database.